### DISCUSSION TITLE
스크린샷 렌더링 퀄리티?

### DISCUSSION THREAD

**Discussion by sunysiny** [naverz/zepeto-studio-kor#912] [5. Q&A]
안녕하세요!
멀티플레이 샘플 프로젝트에 있던 나무와 풀들로 월드맵을 제작하는데
ZepetoScreenShotModule을 사용해서 사진 촬영 후 피드에 게시글로 올리면
퀄리티가 너무 떨어져 보여요
나무와 풀들에 경계선이 지는데 설정을 어떻게 바꿔야 할까요?
포스트프로세싱은 사용 안하고 있어요
![KakaoTalk_20220623_161405963_01](https://user-images.githubusercontent.com/104808306/175248527-c8d95b0b-17af-42e6-b581-a3a30ac6818a.png)
아래는 핸드폰 스크린샷으로 촬영한 사진이에요
![KakaoTalk_20220623_161405963](https://user-images.githubusercontent.com/104808306/175248519-361b4e74-726e-4d2a-8075-56d10f8f3de1.jpg)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요. https://github.com/naverz/zepeto-studio-kor/discussions/634 와 동일한 안티앨리어싱 현상으로 보여집니다.
설정을 No Anti-aliasing 으로 변경하거나 댓글에 있는 코드를 추가해 보세요.
https://github.com/naverz/zepeto-studio-kor/discussions/634#discussioncomment-2613469
감사합니다.
